bazel/api-gen/rendering/entities/categorization.ts

```diff
@@ -102,3 +104,9 @@ export function isSetterEntry(entry: MemberEntry): entry is PropertyEntry {
 export function isDeprecatedEntry<T extends HasJsDocTags>(entry: T) {
   return entry.jsdocTags.some((tag) => tag.name === 'deprecated');
 }
+
+/** Gets whether the given entry is a cli entry. */
+export function isCliEntry(entry: unknown): entry is CliCommandRenderable;
```

Comment:
Is there a way we could make the extracted type for a CLI command a `DocEntry` and the base class for the renderable fit into `DocsEntryRenderable`, or are they incompatibly different?

Reply:
Yeah, the only overlap is `name` and we don't directly explicitly control the CLI json structure since its generated by tooling.